internal/docstore/mongodocstore/codec.go

```diff
@@ -93,15 +94,14 @@ type mapEncoder struct {
 	encoder
 }
 
-func (e *encoder) EncodeMap(n int, isStruct bool) driver.Encoder {
+func (e *encoder) EncodeMap(n int, _ bool) driver.Encoder {
```

Comment:
Can the second argument be removed then?

Reply:
Yes, I added #1969. (I didn't do it in this PR. I have many PRs out and I want to avoid merge conflicts as much as possible.)